### ISSUE TITLE
Travis CI stopped working on rules_scala

### ISSUE THREAD

**Issue by liucijus** [bazelbuild/rules_scala#1138]
Example PR: #1135

**Comment by ittaiz** [comment]
Thanks! @philwo any ideas? Did something change in the org? The webhook is still configured correctly and builds are triggered but it doesn't appear in the checks anymore and I think it's because it doesn't appear in the integrations menu of the repo

**Comment by philwo** [comment]
I would assume that this is related to Travis CI's new pricing model: https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works

I'm investigating now whether there have been any changes to our integrations recently.

**Comment by philwo** [comment]
Seems related: https://travis-ci.community/t/travis-doesnt-report-back-and-all-organization-repos-are-gone/7772/8

**Comment by philwo** [comment]
I signed into Travis and couldn't see any of our organization's repositories. I also wasn't able to add the app to the org (it would only let me add it to my personal account).

I logged out of travis-ci.org and back in again and now I see the builds there. Maybe it works again now?

I would advise to migrate away from Travis CI though. Is there anything missing from Bazel's Buildkite CI that you would need?
